### ISSUE TITLE
nwdiag: ignored shape in group

### ISSUE THREAD

**Issue by dgutson** [plantuml/plantuml#1222]
The `shape` attribute is ignored when first defined in group:

```
@startuml
nwdiag {
  group {
      db01 [shape = "database"];
      db02;
  }
  network internal {
      web01;
      web02;
      db01;
      db02;
  }
}
@enduml
```

However, it works when defined in the `network`:
```
@startuml
nwdiag {
  group {
      db01;
      db02;
  }
  network internal {
      web01;
      web02;
      db01 [shape = "database"];
      db02;
  }
}
@enduml
```

**Comment by The-Lum** [comment]
It works also when you define group after `network`, as:
```puml
@startuml
nwdiag {
  network internal {
      web01;
      web02;
      db01;
      db02;
  }
  group {
      db01 [shape = "database"];
      db02;
  }
}
@enduml
```

**Comment by dgutson** [comment]
Shouldn't work anyways?

**Comment by The-Lum** [comment]
Hi @dgutson, and all,

That is now fixed with:
- https://github.com/plantuml/plantuml/issues/1301#issuecomment-1450504388
- and 690fa34

```puml
@startuml
nwdiag {
  group {
      db01 [shape = "database"];
      db02;
  }
  network internal {
      web01;
      web02;
      db01;
      db02;
  }
}
@enduml
```
>[![](https://www.planttext.com/api/plantuml/svg/SoWkIImgAStDuSehJybCJ5Uevb9GI2_ABoq0iq0W9Sd0K26wE2En85N1La4f9R4aCIcnE5KfrXgXm0Z4hWNYlDIIyloYR8NCl9BKehp47BWfvQb0OwoHE4P8sWqnZQhbIilF1vgXe5gMMbIScP-deSdba9gN0dGg0000)](https://www.planttext.com/?text=SoWkIImgAStDuSehJybCJ5Uevb9GI2_ABoq0iq0W9Sd0K26wE2En85N1La4f9R4aCIcnE5KfrXgXm0Z4hWNYlDIIyloYR8NCl9BKehp47BWfvQb0OwoHE4P8sWqnZQhbIilF1vgXe5gMMbIScP-deSdba9gN0dGg0000)

Thanks to PlantUML team,
Regards.

**Comment by The-Lum** [comment]
Hi all,

_[This is an Issue Review]_
This is now fixed (from V1.2023.5)
- Thanks @dgutson for the report.
- Thanks PlantUML team for your work.

Regards.
